jupyterlab-dataops/Dockerfile

```diff
@@ -23,9 +22,8 @@ RUN code-server --install-extension ms-python.python@${VSCODE_PYTHON} \
 COPY resources/vsc/settings.json /root/.local/share/code-server/Machine/
 COPY resources/vsc/.bashrc root/
 
-# dbeaver
-COPY --from=builder-dbeaver /opt/cloudbeaver/ /opt/cloudbeaver/
-COPY resources/cloudbeaver/conf /opt/cloudbeaver/conf
+# cloudbeaver
+COPY --from=gcr.io/getindata-images-public/cloudbeaver-dataops /opt/cloudbeaver/ /opt/cloudbeaver/
```

Comment:
There is no tag for this image and in line 41 comment suggest that we are using default cloudbeaver and here we are pointing for our custom image with bigquery driver

Reply:
I forgot to pin 0.1.0 version tag here.

Reply:
41 should be in 26 actually, that's missed doubled line